bokeh/util/callback_manager.py

```diff
@@ -49,6 +49,11 @@ def __init__(self, *args, **kw):
         self._event_callbacks = dict()
 
     def on_event(self, event, *callbacks):
+        ''' Run callbacks when the specified event occurs on this Model
+
+        Not all ``Event``s are supported for all ``Model``s.
+        See specific ``Event``s in ``bokeh.events`` for more information on which ``Model``s can trigger them.
```

Comment:
Sphinx is getting confused, and also it's harder to read as text, so I'd suggest removing all the backticks above. A nice touch would also make ``bokeh.events`` be a link using a Sphinx reference 
```
:ref:`bokeh.events`
```
Lastly although it's not 100% enforced we do try to keep docstrings in particular to about 80 characters wide.